README.md

```diff
@@ -10,8 +10,8 @@ bower install --save rss-parser
 
 ## Usage
 You can parse RSS from a URL, local file (NodeJS only), or a string.
-* `parseString(xml, callback)`
-* `parseFile(filename, callback)`
+* `parseString(xml, [settings,],  callback)`
+* `parseFile(filename, [settings,], callback)`
```

Comment:
Also, can you add a couple lines documenting `options.customFields.item` and `options.customFields.feed`?